development.md

```diff
@@ -0,0 +1,156 @@
+# Development
+
+## Introduction
+
+OpenRazer consists of a Linux kernel module and Python daemon + library. The Linux kernel module exposes an
+sysfs interface, the Python daemon (openrazer-daemon) exposes this on it's turn over D-Bus.
+
+If you want to add support for a new hardware device it's often required to make changes to both the kernel module
+as the Python sources. For example `io.github.openrazer.openrazer.metainfo.xml` and `99-razer.rules`. The
+fake drivers can be regenerated by running `./scripts/generate_all_fake_drivers.sh -f`.
```

Comment:
I'd remove these two sentences here, not really relevant in the introduction.

> For example `io.github.openrazer.openrazer.metainfo.xml` and `99-razer.rules`. The
fake drivers can be regenerated by running `./scripts/generate_all_fake_drivers.sh -f`.

Reply:
It was suggested by @lah7 ... Please advice

Reply:
My suggestion was just trying to "document" which files need updating that might be missed when _hacking_ locally.

I just thought it would be useful to mention them - it's obvious to us, but maybe not new contributors:

* `io.github.openrazer.openrazer.metainfo.xml` - Metadata for repositories (appstream). Add new PID here.
* `99-razer.rules` - Set permissions for the driver at start up. Add new PID here also.
* Fake driver is generated automatically, don't create files manually.

If it's not valuable, pretend I never suggested this :)